components/layout_2020/style_ext.rs

```diff
@@ -583,6 +583,7 @@ impl ComputedValuesExt for ComputedValues {
 
     /// Whether or not the `overflow` value of this style establishes a scroll container.
     fn establishes_scroll_container(&self) -> bool {
+        // Checking one axis suffice
```

Comment:
nit: suffices

Reply:
```suggestion
        // Checking one axis suffices, because the computed value ensures that
        // either both axes are scrollable, or none is scrollable.
```